app/tests/test_token_repository.py

```diff
@@ -0,0 +1,67 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+import boto3
+from docker import from_env, DockerClient
+from docker.models.containers import Container
+
+
+class TestTokenRepository(TestCase):
+    dynamodb_container: Container = None
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.docker_client = from_env()
+        cls.dynamodb_container_name = 'dynamo-test'
+        cls.dynamodb_container = _run_dynamodb_container(cls.docker_client, cls.dynamodb_container_name)
+        try:
```

Comment:
I would encapsulate this try/except in a "setup_hallebarde_database" function, what do you think ?